GeeksCoreLibrary/Core/Services/WiserItemsService.cs

```diff
@@ -2424,14 +2424,20 @@ public async Task<List<WiserItemModel>> GetLinkedItemDetailsAsync(IWiserItemsSer
                 where.Add("(permission.id IS NULL OR (permission.permissions & 1) > 0)");
             }
 
+            var tablePrefixItemId = "";
+            if (linkSettings.UseItemParentId && reverse && !string.IsNullOrWhiteSpace(itemIdEntityType))
+            {
+                tablePrefixItemId = await wiserItemsService.GetTablePrefixForEntityAsync(itemIdEntityType);
+            }
+            
             var tablePrefix = "";
-            if (!String.IsNullOrWhiteSpace(entityType))
+            if (!string.IsNullOrWhiteSpace(entityType))
             {
                 tablePrefix = await wiserItemsService.GetTablePrefixForEntityAsync(entityType);
                 databaseConnection.AddParameter("entityType", entityType);
                 where.Add("item.entity_type = ?entityType");
             }
-            else if (!String.IsNullOrWhiteSpace(itemIdEntityType))
+            else if (!string.IsNullOrWhiteSpace(itemIdEntityType))
```

Comment:
please use String.IsNullOrWhiteSpace instead of string.IsNullOrWhiteSpace

Reply:
@MikeVanMourik , but Rider says this. When using a lower case s, then it doesn't give suggestions.

Reply:
Thats something about the style settings of Rider then, I dont really care what the Linter suggests

if i search through the geeks core project and for string.IsNullOrWhiteSpace and make sure the case needs to match i only get one result, using String.IsNullOrWhiteSpace is the standard here.

if we want to change that its fine but thats outside the scope of this pull request.

Reply:
@rvdgraaf Your Rider has different settings than ours. So you can either import our Rider settings (which are saved on Google drive and in Nuclino I believe), or keep your own settings and ignore warnings when they're different than our coding guidelines.